src/main/java/net/tropicraft/core/common/dimension/feature/jigsaw/PathStructureProcessor.java

```diff
@@ -1,83 +1,84 @@
-package net.tropicraft.core.common.dimension.feature;

```

Comment:
Line ending changes?

Reply:
File moved packages, not really sure why it said the whole file changed, that's annoying.

Reply:
The others that moved didn't get this, my guess is the line endings got changed. Or some other whitespace change.